types/aframe/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "2.3.2",
+  "compilerOptions": {
+    "baseUrl": "..",
+    "lib": [
+      "es5",
+      "dom",
+      "es2015.iterable",
+      "es2015.promise"
+    ],
+    "module": "commonjs",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "strictNullChecks": true,
+    "noEmit": true,
+    "forceConsistentCasingInFileNames": true,
+    "typeRoots": [ "../" ],
+    "types": [ ]
+  },
+  "files": [
+    "index.d.ts",
+    "aframe-tests.ts"
```

Comment:
Either include your `tests` folder here or remove them all together, this way the CI would verify that changes to the declaration file does not break the tests.